CiarnuroRT/webapp/style.css

```diff
@@ -234,9 +234,9 @@ input[type="button"]:active{
 div.list > div.entry{
     display:flex;
 	align-items:center;
-	min-height:5em;
+	min-height:6rem;
 	width:100%;
-	border-bottom:1px solid #CCCCCC;
```

Comment:
px can be used to make super thin borders, 0.1rem is thicker

Reply:
I resolve that thing, i thought that px is not allowed to be used so initially i changed with rem